internal/v4/archive_test.go

```diff
@@ -1532,12 +1531,13 @@ func patchArchiveCacheAges(s interface {
 // whether the id in the request was fully specified.
 // It assumes that patchArchiveCacheAges has been called
 // for the current test.
-func assertCacheControl(c *gc.C, h http.Header, idFullySpecified bool) {
-	seconds := 5
-	if idFullySpecified {
-		seconds = 20
+func assertCacheControl(c *gc.C, h http.Header, isPublic bool) {
+	if isPublic {
+		seconds := 5
+		c.Assert(h.Get("Cache-Control"), gc.Equals, fmt.Sprintf("public, max-age=%d", seconds))
+	} else {
```

Comment:
normally we would save the indent by returning in the clause above and not having the else.